lib/PlaidClient.js

```diff
@@ -210,6 +211,44 @@ Client.prototype.getTransactions =
     }, options, cb);
   };
 
+// getAllTransactions(String, Date, Date, Object?, Function)
+Client.prototype.getAllTransactions =
+  function(access_token, start_date, end_date, cb) {
+    var transactions = [];
+    var offset = 0, total_transactions = null;
```

Comment:
nit: two statements per line

we don't have a linter set up for this project, but it would probably not like this :).